### PR TITLE
Refactor element hiding content script to be an es6 class

### PR DESCRIPTION
**Pull request by dharb**
**Reviewer:** @kdzwinel 

## Description:
No functionality change here, just refactoring to make the element hiding content script an ES6 class.

## Steps to test this PR:
1. Make sure element hiding works exactly as it did before. Note that not all of the ads on the site we've been testing this on (yahoo.com) will be collapsed since they don't all contain blocked tracking requests.

### CODE REVIEW

File: shared/js/content-scripts/element-hiding.js [hunk 1]
```diff
@@ -7,124 +7,131 @@
...
+                this.hideFrame(frame.parentNode)
             }
-        })
-    } else {
-        chrome.runtime.sendMessage({hideElements: true, url: document.location.href, frame: contentScript.frameType})
+        }
     }
+
+    // Instantiate content script
+    var contentScript = new ContentScript()
```
Comment (dharb) [inline shared/js/content-scripts/element-hiding.js]: eslint is throwing a no-unused-var error here - https://travis-ci.org/duckduckgo/duckduckgo-privacy-extension/builds/408297396.

**Comment by andrey-p** [comment]
Just a note, I don't know if content scripts are ever transpiled. Is that something we're worried about?

**Comment by andrey-p** [comment]
@kdzwinel Cool, thanks. Just don't want to get caught off guard by something on some slightly older browser that people are somehow still using.

**Comment by dharb** [comment]
Thanks @kdzwinel! That makes perfect sense to me, good call with those changes.

**Pull request merged**